src/components/study-pane.js

```diff
@@ -956,6 +1054,20 @@ const StudyPane = (props) => {
                             computationStopped={computationStopped}
                         />
                     </div>
+                    {lineMenu.display && (
```

Comment:
why use `lineMeny.display &&` instead of using the "open" prop of the menu?

Reply:
LineMenu should not be rendered (even with an "open" prop to false)  if no line is selected (i.e. if lineMenu.display is false) because menu items labels depend on the selected line properties.